OpenData/secular printed material - 1921 - 1990/Coraa ny Gael/Coraa ny Gael 1953-09-15/document.csv

```diff
@@ -1,16 +1,16 @@
 ﻿Manx,English,Notes
 CORAA NY GAEL,The Voice of the Gaels,
-"     Ta Mnr. R. H. Kinvig[1], olloo dy cruinnahushtey ’syn ard-scoail [sic] Birmingham, ’syn ellan nish. Failt ort y charrey","     Mr. R. H. Kinvig, professor of geography in Birmingham University, is on the Island now. Welcome, friend.","[1] R. H. Kinvig] 1894-1969. Professor of Geography at Brimingham University. Author of ‘A History of the Isle of Man’ Oxord, 1945., and ‘The Isle of Man: A Social, Cultural and Political History’, Liverpool University Press, 1975."
+"     Ta Mnr. R. H. Kinvig[1], olloo dy cruinnahushtey ’syn ard-scoail [sic] Birmingham, ’syn ellan nish. Failt ort y charrey","     Mr. R. H. Kinvig, professor of geography in Birmingham University, is on the Island now. Welcome, friend.","[1] R. H. Kinvig] 1894-1969. Professor of Geography at Birmingham University. Author of ‘A History of the Isle of Man’ Oxford, 1945., and ‘The Isle of Man: A Social, Cultural and Political History’, Liverpool University Press, 1975."
 "     Hie Mnr. as Bnr. D. F. Corlett jeh Yohannesburg, er-ash hug Yn Africa Jiass laa jea. T’ad er ve ceau mee er y Twoaie vooar, ayns Rhumsaa.",     Mr and Mrs D. F. Corlett of Johannesburg went back to South Africa yesterday. They have spent a month in the ‘Great North’ in Ramsey.,
 "     Ta Coyrleder Frank Corkill er coural lurg y chingys v’echey. Heb y Meoir, Coyrleder Joseph Cain, oltaghey da tra haink eh er-ash da’n Cooinseil Ghoolish er y chiaghtyn shoh chaie.","     Councilor Frank Corkill has recovered after the sickness he has had. The mayor, councilor Joseph Cain offered a welcome to him when he reappeared for Douglas Council this last weekend.",
-"Va Mnr. Juan Thobm Kaighin, Balleygarret, Skyll Vreeshey, kaire feed blein as ’nane jeig d’eash er y cheid laa jeh’n vee shoh ! ","     Mr. John Tom Kaighin, Ballagarret, Bride, was ninety-one years old on the first of this month! ",
+"Va Mnr. Juan Thobm Kaighin, Balleygarret, Skyll Vreeshey, kaire feed blein as ’nane jeig d’eash er y cheid laa jeh’n vee shoh! ","     Mr. John Tom Kaighin, Ballagarret, Bride, was ninety-one years old on the first of this month! ",
 Ta mee sheiltyn d’el eh-hene gaase ny ’saa gagh vlein!,I guess that he himself is growing younger every year!,
-"Ta shin ooilley fo lhiastnys vooar dasyn son ooilley’n Ghailck vie, skeeaylyn as bealedys [beealydys], creenaght as tushtey, ta, dy-kinjagh deayrtey magh ass y veal echey goll-rish struan!","We are all in great debt to him for all the good Manx, stories and folklore, wisdom and knowledge, that always flows forth from his mouth like a stream!",
+"Ta shin ooilley fo lhiastnys vooar dasyn son ooilley’n Ghailck vie, skeeaylyn as bealedys [beealydys], creenaght as tushtey, ta, dy-kinjagh deayrtey magh ass y veal echey goll-rish struan!","We are all in great debt to him for all the good Manx stories and folklore, wisdom and knowledge, that always flows forth from his mouth like a stream!",
```

Comment:
This is a translation from:

> Ghailck vie, skeeaylyn as bealedys

"good Manx" is a separate element in the list

```suggestion
"Ta shin ooilley fo lhiastnys vooar dasyn son ooilley’n Ghailck vie, skeeaylyn as bealedys [beealydys], creenaght as tushtey, ta, dy-kinjagh deayrtey magh ass y veal echey goll-rish struan!","We are all in great debt to him for all the good Manx, stories and folklore, wisdom and knowledge, that always flows forth from his mouth like a stream!",
```

Reply:
Why am I making so many mistakes with this proofreading? I do this all the time, and I'm fairly sure I don't misread things this often normally. Sorry!

Reply:
I wouldn't worry too much about it, Jane, probably the extra brain power of looking at the Manx, even out of the corner of your eye, makes it just a little bit more tricky.

Ken

Reply:
True!